tools/pull_requests/sme_groups.json

```diff
@@ -0,0 +1,43 @@
+{
+    "APIs": [],
+    "Backup": [],
+    "Block cache": [],
+    "Block manager": ["etienneptl"],
```

Comment:
I'm keeping this barebones to start with. When the bot's online properly I'll reach out to the team to see who'd like to be included.